Common/src/main/resources/assets/tipsmod/lang/en_us.json

```diff
@@ -70,6 +70,18 @@
   "tipsmod.tip.freezing_leather": "Leather armor can protect against freezing damage.",
   "tipsmod.tip.azalea_caves": "Azalea trees grow above the lush cave biomes.",
 
+  "tipsmod.tip.joke.button": "Despite the name, you don't have to sit on Buttons for them to work",
+  "tipsmod.tip.joke.combat": "Zombies dislike it when you stab them with a sword.",
+  "tipsmod.tip.joke.swim": "If you stay under water too long you may drown",
+  "tipsmod.tip.joke.hop": "Despite the name, Hoppers aren't related to Rabbits.",
+  "tipsmod.tip.joke.lava": "Like any items, buckets will burn in lava, but not when it is IN the bucket.",
+  "tipsmod.tip.joke.stairs": "Placing stairs upside down does not allow you to walk on ceilings.",
+  "tipsmod.tip.joke.skulls": "Being ill tempered, Wither Skeletons lose their head easily.",
+  "tipsmod.tip.joke.creeper": "Creepers can become Thunderstruck. Stay away from AC/DC near them",
```

Comment:
This sentence is a bit weird in english. Maybe "Don't play AC/DC near Creepers, they can become Thunderstruck.